src/js/components/accordion.js

```diff
@@ -20,35 +30,51 @@ export default class Accordion {
 
     this.triggerAttribute = 'data-accordion-trigger';
     this.triggerSelector = `[${this.triggerAttribute}]`;
-    this.triggers = nodeListToArray(this.element.querySelectorAll(this.triggerSelector));
+    this.triggers = nodeListToArray(
+      this.element.querySelectorAll(this.triggerSelector)
+    );
     if (this.triggers.length < 2) {
       // eslint-disable-next-line no-console
-      throw new TypeError('An accordion should contain *at least two* accordion triggers with the "data-accordion-trigger" attribute');
+      throw new TypeError(
+        'An accordion should contain *at least two* accordion triggers with the "data-accordion-trigger" attribute'
+      );
     }
     this.panelAttribute = 'data-accordion-panel';
     this.panelSelector = `[${this.panelAttribute}]`;
-    this.panels = nodeListToArray(this.element.querySelectorAll(this.panelSelector));
+    this.panels = nodeListToArray(
+      this.element.querySelectorAll(this.panelSelector)
+    );
 
     // Set this.openOnInit if needed
     try {
-      this._quantifySelector(this.element, `[${'data-accordion-panel-init'}]`, () => {
-        let initialPanel;
+      let excess;
 
-        // Determine if a specific panel has been initialized with the data-tab-init attribute, otherwise, use the first tab
-        this.panels.forEach((panel, index) => {
-          if (panel.hasAttribute('data-accordion-panel-init')) {
-            initialPanel = panel;
-          } else {
-            this.panels[index].setAttribute('hidden', '');
-          }
-        });
+      this.element.querySelectorAll(`[${'data-accordion-panel-init'}]`).length >
+      1
+        ? (excess = true)
+        : (excess = false);
+
+      if (excess === true) {
+        throw new TypeError('Caught');
+      }
```

Comment:
This conditional variable setting using the ternary based on the number of elements with the `data-accordion-panel-init` could be simplified a lot by querying in `this.element`, storing that in a variable, and then checking that in your `if` statement. You could eliminate the ternary all together.

```js
const initializedPanels = this.element.querySelectorAll(
  `[${"data-accordion-panel-init"}]`
);

if (initializedPanels.length > 1) {
  throw new TypeError('Caught');
}
```

Reply:
Also, just and FYI, if you're not interpolating any data into this template literal (backticks) statement, you don't need to use the `${}` syntax here to wrap the `data-accordion-panel-init` attribute. 

So it should be the regular attribute selector:
```js
const initializedPanels = this.element.querySelectorAll(
  `["data-accordion-panel-init"]`
);
```